app/assets/stylesheets/application.scss

```diff
@@ -144,3 +144,7 @@ $u-can-act-grey: #99b5cd;
   z-index: 999;
   transition: all .1s ease-in-out;
 }
+
+.cookie-warning{
+  display: none;
+}
```

Comment:
Is het ook mogelijk om dit gewoon rechtstreeks in het style attribuut van de cookie-warning div in de application layout template te zetten? Want nu is het een CSS+cookie test en dan is het alleen een cookie test.

(ik zat te denken ook als de css wat traag ingeladen wordt, dattie dan misschien op sommige browsers even kort die cookie-warning div laat zien voordat de css m verbergt.)